Flow.Launcher.Infrastructure/Image/ImageCache.cs

```diff
@@ -66,8 +66,10 @@ public void Initialization(Dictionary<string, int> usage)
                 {
                     // To delete the images from the data dictionary based on the resizing of the Usage Dictionary.
 
-
-                    foreach (var key in Data.OrderBy(x => x.Value.usage).Take(Data.Count - MaxCached).Select(x => x.Key))
+                    foreach (var key in Data.Where(x => x.Key != Constant.MissingImgIcon)
+                                            .OrderBy(x => x.Value.usage)
+                                            .Take(Data.Count - MaxCached)
+                                            .Select(x => x.Key))
                     {
                         if (!(key.Equals(Constant.ErrorIcon) || key.Equals(Constant.DefaultIcon)))
```

Comment:
Question: Should we keep this two here in resizing check? I think the ErrorIcon should be replaced with MissingIcon, though.

Reply:
Agree, I think we should replace error with missing. Ideally missing img usage should be low, but I still see it often when using Explorer plugin, so since missing and default imgs are commonly used, we should continue to exclude them from removal. I will make a separate pr just to keep it clean.

Reply:
I think since they are commonly used, their usage won't be low so that they won't be removed. If they are low, then just removing them won't cause some issue.
Btw, I think we should use Except in Linq to except the key that we don't want to remove, which seems more clear.